Femlives/femlives
# Feat/fl 44/send email flow

## Description

- adds first resend infrastructure
- adds a first email template for proof of concept
- adds first submit action which saves the email address in the db and sends a confirmation mail
- adds util function to assert strings

**!!! IMPORTANT !!!!**
You need to add the resend environment variable to your env.local in order to make it work

## How Can This Be Tested?

run `npm i`

Quick and dirty:
you can add the following code to page.tsx and run it
errors could be popping up but it should work nevertheless

```ts
  useEffect(() => {
    const handleSubmit = async () => {
      await submitEmailSignUp({ email: [YOUR EMAIL] });
    };
    handleSubmit();
  }, []);
```

## Checklist

- [x] 🙅‍♀️ No leftover console.logs
- [x] 🦾 Sufficient tests, e. g. unit test
- [x] 🧠 Left over TODOs are marked with a url to the ticket
